CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 2.1.0 (in progress)
 **SDK**
 - Support telephone hyperlink in miniapp. See [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).
+- Support webview video fullscreen.
```

Comment:
Would it require any support from the hosting container?

Reply:
Do you mean web hosting or hostapp? Both are none.

Reply:
Yea, ideally listing the dependencies in `userguide`. There were flags added to the display activity. So, probably the same flags are needed on the parent container?

Reply:
No, there is no flag here. Display in fullscreen is default.